.gitignore

```diff
@@ -9,3 +9,16 @@ erl_crash.dump
 
 # We have example files for docker-compose.yml so let's ignore it
 docker-compose.yml
+
+/node_modules
+
+# Ignore in umbrella apps node modules
+apps/*/node_modules/
+
+# Ignore generated files of in umbrella apps
```

Comment:
@edipox what about this? ... I think that instead of trying to match all possible sub directories of **apps/** we can let every child app declare its own **.gitignore** file. Do you know if that's possible using git?